Logs Landing Page

rachelwhitton opened the pull request:
Connects to #359 

- Create logs landing page
- Create logs directory for breadcrumbs
- Moves the following and updates links throughout:
 - source/docs/articles/sites/debugging-sites-with-log-files.md -> source/docs/articles/sites/logs/debugging-sites-with-log-files.md
 - source/docs/articles/sites/downloading-live-error-logs.md -> source/docs/articles/sites/logs/downloading-live-error-logs.md
 - source/docs/articles/sites/code/php-slow-log.md -> source/docs/articles/sites/logs/php-slow-log.md
 - source/docs/articles/sites/database/mysql-slow-log.md -> source/docs/articles/sites/logs/mysql-slow-log.md

Linkchecker has been run with all 404s resolved.

@bmackinney and @nataliejeremy can you review? Leaving the issue open - planning to add individual PR's for each article created.

rachelwhitton commented:
Todo:

- [ ] Rebase to master and update link created in #752 once it has been merged.

rachelwhitton commented:
@timani, @bmackinney can you review and provide feedback on this PR and related #737?

timani commented:
Ship it, we can always iterate

ari-gold commented on source/docs/articles/sites/logs.md:
```diff
@@ -0,0 +1,67 @@
+keywords: log, access log, nginx access log, nginx log, nginx access, nginx error, nginx error log, php error, php error log, php fpm error, php fpm, php error, php slow, php slow log, pyinofity log, pyinofity, watcher log, watcher
+draft: true
+---
+Log files track and record your site's activity which help you find, debug, and isolate current or potential problems on your site. These logs can also be used to compile statistics on your visitors and referral traffic.
```
I think we should remove the 2nd sentence, because any traffic hitting Varnish won't hit the application container's nginx so won't be recorded in nginx-access.log

rachelwhitton replied:
Great point @ari-gold - thanks!

ari-gold commented on source/docs/articles/sites/logs.md:
````diff
@@ -0,0 +1,67 @@
+    └──watcher.err
+    └──watcher.log
+```
+You can automate this process with the following script:
````
Can we remove lines 35-45 and just link to the other doc? Or keep it here and not have the separate doc.

ari-gold commented on source/docs/articles/sites/logs.md:
````diff
@@ -0,0 +1,67 @@
+get logs/mysqld-slow-query.log
+```
+You now have a local copy of the `mysqld-slow-query.log` file.
+## See Also
````
Link to New Relic doc

ari-gold commented:
+1

rachelwhitton commented:
@ari-gold I think i'm using `rebase` incorrectly, i'm getting duplicate commits.

My exact commands were:
`git rebase master`
Then I incorporated your comments into the doc, staged and committed. 
```
git push origin 359-log
! [rejected]        359-log -> 359-log (non-fast-forward)
error: failed to push some refs to 'https://github.com/pantheon-systems/documentation.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
So then I pulled down, which created a conflict. So i fixed and pushed up - but these duplicate commits are the opposite of what we're trying to accomplish (cleaner commits into master). Any tips? After a bit of googling it seems I should be force pushing to the feature branch?
